src/chapters/01-orientation.md

```diff
@@ -145,19 +161,46 @@ ember server | awk '{ \
 ember server
 ```
 
+The development server is responsible for compiling our app and serving it to the browsers. It may take a while to boot up, but once it's up an running, open your favorite browser and head to http://localhost:4200. You should see the following welcome page:
```

Comment:
```suggestion
The development server is responsible for compiling our app and serving it to the browsers. It may take a while to boot up. Once it's up and running, open your favorite browser and head to http://localhost:4200. You should see the following welcome page:
```